src/Symfony/Component/Debug/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.2.0
```

Comment:
that'll be in 4.3 now

Reply:
Do you mean it's too late already for this depreciation to be included in SF 4.2 ?

Reply:
yes that's what I mean, 4.2 is in feat freeze since the end of December

Reply:
end of September actually :smile: